pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,23 +125,23 @@ public void testBackLogout() {
         casClient.setCallbackUrl(CALLBACK_URL);
         casClient.init(null);
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
-            .setRequestMethod(HTTP_METHOD.POST);
+            .setRequestMethod("POST");
```

Comment:
I think you could keep the enum (or maybe some constants) for all the cases where in pac4j code you need to refer to an HTTP method instead of using a plain string.